content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+
+The access token have a limited lifetime (about 720 seconds or 2 hours). Once it expires, the client application must use the obtained client-id and client-secret to generate a new access token. One indication of the expiration of access-token, the request to DSCC API will return response error: '401 Unauthorized HTTP.'  If the client application generates new access token prior to it expires, then the current access-token will be invalidated or treated as not authorized. 
+
+Additionally, a user can also change the client-secret to update the authorization when the authorized user has lost the client-secret or the secret has been compromised. 
+
+And lastly, when access to the API must be disabled, a user can delete the API client credential associated with client-id and client-secret in the API Gateway.
+
+![Access API process](/img/user-guide-for-authorization.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, the user must log into the [HPE GreenLake Console,](https://common.cloud.hpe.com) have deployed the DSCC to the intended region, and onboarded a storage array (HPE Alletra, HPE Nimble, or HPE Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. For instance, the user must have volume management capability in the Data Ops Management to create a storage volume in US region. For more information about the role based access control, please take a look at the [HPE GreenLake User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=ccs-help_en_us)
+
+The API Gateway menu is available inside the Manage menu. From HPE Greenlake Console click on Menu to access this Manage Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway menu provides the following operations:
+
+1. Create and manage API client credential association in order to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals).
+   2. Client ID (Hexadecimals).
+   3. Client Secret (Hexadecimals).
+   4. URL to the HPE GreenLake Console API end-point (string).
+   5. URL to the HPE DSCC API end-point (string).
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each instance of the API client credential represents the authorization relationship between the client application and the DSCC REST API end point. Click on the Create Credentials button to generate a client credential. Afterward the user can obtain the client secret and client secret, and used them to generate the access token.
```

Comment:
Last sentence, change "Afterward" to "Afterwards," and change "used" to "use"
You have two "client secret" here... Did you mean "Client ID" and "Client Secret"? Be consistent with you capitalization.